Stylesheets/common-css/main.css

```diff
@@ -0,0 +1,270 @@
+.page {
+    background-color: var(--st-white);
+}
+
+.page-container {
+    min-width: 83%;
+    max-width: 87%;
+    margin: auto;
```

Comment:
Instead of doing like this that is specifying both max-width and min-width, use combination of width and max-width

```css
width: 90vw; //width will be 90vw width
max-width: 1024px; // width will not exceed this value even if the website is viewed in ultrwide monitors
margin: 0 auto; //centers the content. 
```

For now it is fine. Change  it later on.